lib/service/connect.go

```diff
@@ -827,7 +827,7 @@ func (process *TeleportProcess) findReverseTunnel(addrs []utils.NetAddr) (string
 			if resp.Proxy.SSH.TunnelPublicAddr != "" {
 				return resp.Proxy.SSH.TunnelPublicAddr, nil
 			}
-			return resp.Proxy.SSH.TunnelListenAddr, nil
+			return "auth:3024", nil
```

Comment:
Should this be a constant/default?

Also, due to this change the comment above is no longer valid.

Reply:
sorry, that was a leftover from my local testing, removed it.